polyfill/test/Absolute/constructor/from/subclass-invalid-receiver.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
```

Comment:
Is the name of this test intentional? In some of the other tests it's called "branding", and this one doesn't seem to have to do with subclasses (although it does test for invalid receivers).

(Ditto for several other tests below)